source/includes/cloudstack/_bare_metal_instances.md

```diff
@@ -0,0 +1,82 @@
+### Bare metal instances
+
+Deploy and manage your bare metal instances.
+
+For information regarding virtual instances, please see [instances](#cloudstack-instances).
+
+#### List bare metal instances
+
+Identical to virtual instances, please see [list instances](#cloudstack-list-instances).
+
+#### Retrieve a bare metal instance
+
+Identical to virtual instances, please see [retrieve an instance](#cloudstack-retrieve-an-instance).
+
+#### Acquire a bare metal instance
+
+```shell
+curl -X POST \
+   -H "Content-Type: application/json" \
+   -H "MC-Api-Key: your_api_key" \
+   -d "request_body" \
+   "https://cloudmc_endpoint/v1/services/compute-on/test_area/instances"
+
+# Request should look like this
+```
+
+```json
+{
+   "name": "jarvis",
+   "computeOfferingId": "3caab5ed-b5a2-4d8a-82e4-51c46168ee6c",
+   "networkId": "55ccea7f-8286-479e-a648-dd4a45866daf",
+   "portsToForward": ["80", "9999"],
+   "sshKeyName": "mysshkey",
+   "templateId": "5f968ad6-56d0-4d0d-ad7e-f8f4a5b5d986",
+   "userData": "#!/bin/bash\necho 'hello world'"
+}
+```
+
+<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/instances?operation=acquireBareMetal</code>
+
+Acquire a bare metal instance in an [environment](#administration-environments). This endpoint allows you to easily add [port forwarding rules](#port-forwarding-rules) and user data to the instance without doing additional API calls. This operation is analogous to the [create an instance](#cloudstack-create-an-instance) operation for virtual instances.
+
+Required | &nbsp;
+------ | -----------
+`name`<br/>*string* | Name of the newly created instance
+`templateId`<br/>*UUID* | The [template](#cloudstack-templates) to use for this instance.
+`computeOfferingId`<br/>*UUID* | The [compute offering](#cloudstack-compute-offerings) will determine the hardware specifications of your instance.
+`networkId`<br/>*UUID* | The [network](#cloudstack-networks) in which the instance will be created. If you don't have a network, it can be created through the [create network](#cloudstack-create-network) api.
+`sshKeyName`<br/>*string* | The name of the [SSH key](#cloudstack-ssh-keys) to use for this instance. If you don't have an SSH key registered, you can do so through this [api](#cloudstack-create-ssh-key).
+
```

Comment:
We used to sort them alphabetically...no?

Reply:
Done